### PR TITLE
AI preso 2 outline for team Roguelike

### PR DESCRIPTION
**Pull request by keyonzi**
Here is our outline, sorry it was a bit late.

### CODE REVIEW

File: adv_topics/ai.md [hunk 1]
```diff
@@ -33,12 +33,27 @@
...
-* Topic 1
-	* Details
-	* Details
-	* Details
-* Topic 2
-	* Details
-	* Details
-	* Details
+* Introduction (10 min)
+	* General introduction of what it is
```
Comment (nfarnan) [inline adv_topics/ai.md:37]: Already covered in previous presentation, you should build on top of what they presented, not cover the same ground.

File: adv_topics/ai.md [hunk 2]
```diff
@@ -33,12 +33,27 @@
...
+	* History of AI
+		* Mario AI, AI in its simplest form
```
Comment (nfarnan) [inline adv_topics/ai.md:39]: Again, history was already covered. I'll allow this if it differs from previous presentation AND has some impact on your approach to your game. Be sure to contrast this with the history covered in the first presentation, though.

File: adv_topics/ai.md [hunk 3]
```diff
@@ -33,12 +33,27 @@
...
+	* Psuedointelligence vs artificial intelligence
+		* Psudeointelligence mimics what we think of AI, it gives enemies and NPCs
+			directions, giving the appearance of AI
+		* 'True' AI actually learns from the game itself and the player to improve itself		
+	* Different types of AI (quick introduction)
+		* Stateless AI
+		* State Machine AI
+		* Neural Networks
+			* Deep Learning and the training process to improve in-game AI
+* Topic 1 - Deep Dive of State Machine vs Machine Learning (Neural Networks)
```
Comment (nfarnan) [inline adv_topics/ai.md:50]: You annotated 10 min for intro (too much, should definitely be cut down for presentation 2). How much time for these other topics?

**Review by nfarnan — CHANGES_REQUESTED** [review]
Need more details on timing, also need to cut down the intro given overlap with material from the first AI presentation.

File: adv_topics/ai.md (hunk 2 above)
Comment (keyonzi) [reply adv_topics/ai.md]: We cut out the history, and kept only a general intro as a reminder. Instead our 'intro' will focus on difference between psuedointelligence and AI, and what that actually means for games. Plus we will introduce different mechanics that the other group did not.

File: adv_topics/ai.md (hunk 3 above)
Comment (keyonzi) [reply adv_topics/ai.md]: Cut intro to 5. But included 5 for the breakdown of new types of AI. The introduction of the other AI methods (stateless, deep learning) will actually take some time. But in my fork we made that more clear.

**Comment by keyonzi** [comment]
I see 'changes' requested. I commented on all of your questions. But I also updated the whole file on my fork to reflect what I said in my comments. But the option for a PR isn't there, because this one is outstanding I think. But you can look there for more details on the changes.

https://github.com/keyonzi/cs1666_organization/blob/main/adv_topics/ai.md

**Review by nfarnan — APPROVED** [review]

**Pull request merged**